web/src/components/onboarding/LockedProfileField.tsx

```diff
@@ -12,12 +12,17 @@ export const LockedProfileField = (props: Props): ReactElement => {
   const { state } = useContext(ProfileDataContext);
 
   let valueToDisplay = state.profileData[props.fieldName] as string;
+
+  if (props.fieldName === "municipality") {
```

Comment:
I'll be honest, I don't love this approach because I don't love the hardcoding of specific field names and their specialized logic into an if condition.

However, I can't think of a solution right now that's better. so 🤷